src/bo4e/bo/ausschreibung.py

```diff
@@ -0,0 +1,99 @@
+"""
+Contains Ausschreibung class and corresponding marshmallow schema for de-/serialization
+"""
+from datetime import datetime
+from typing import List, Optional
+
+import attr
+from marshmallow import fields
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.bo.geschaeftsobjekt import Geschaeftsobjekt, GeschaeftsobjektSchema
+from bo4e.bo.geschaeftspartner import Geschaeftspartner, GeschaeftspartnerSchema
+from bo4e.com.ausschreibungslos import Ausschreibungslos, AusschreibungslosSchema
+from bo4e.com.zeitraum import Zeitraum, ZeitraumSchema
+from bo4e.enum.ausschreibungsportal import Ausschreibungsportal
+from bo4e.enum.ausschreibungsstatus import Ausschreibungsstatus
+from bo4e.enum.ausschreibungstyp import Ausschreibungstyp
+from bo4e.enum.botyp import BoTyp
+from bo4e.validators import check_list_length_at_least_one
+
+
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
+@attr.s(auto_attribs=True, kw_only=True)
+class Ausschreibung(Geschaeftsobjekt):
+    """
+    Das BO Ausschreibung dient zur detaillierten Darstellung von ausgeschriebenen Energiemengen in der Energiewirtschaft
+    """
+
+    # required attributes
+    bo_typ: BoTyp = attr.ib(default=BoTyp.AUSSCHREIBUNG)
+    #: Vom Herausgeber der Ausschreibung vergebene eindeutige Nummer
+    ausschreibungsnummer: str = attr.ib(validator=attr.validators.instance_of(str))
+    #: Aufzählung für die Typisierung von Ausschreibungen
+    ausschreibungstyp: Ausschreibungstyp = attr.ib(validator=attr.validators.instance_of(Ausschreibungstyp))
+    #: Bezeichnungen für die Ausschreibungsphasen
+    ausschreibungsstatus: Ausschreibungsstatus = attr.ib(validator=attr.validators.instance_of(Ausschreibungsstatus))
+
+    #: Kennzeichen, ob die Ausschreibung kostenpflichtig ist
+    kostenpflichtig: bool = attr.ib(validator=attr.validators.instance_of(bool))
+    #: Gibt den Veröffentlichungszeitpunkt der Ausschreibung an
+    veroeffentlichungszeitpunkt: datetime = attr.ib(validator=attr.validators.instance_of(datetime))
+
```

Comment:
I'm not sure myself what looks nicer here. For the other objects we have always listed the attributes without an empty line.
```suggestion

```